src/key.js

```diff
@@ -1415,13 +1553,14 @@ function getExpirationTime(keyPacket, signature) {
 
 /**
  * Returns the preferred signature hash algorithm of a key
- * @param  {object} key
+ * @param  {module:key.Key} (optional) key
+ * @param  {module:packet.SecretKey|module:packet.SecretSubkey} keyPacket
```

Comment:
Can you write a brief explanation of the purpose of the variable? Like "key packet used for signing" or something like that